backend/src/utils/componentUtils.ts

```diff
@@ -203,6 +203,12 @@ const getCSVForApp = (
         return csv;
       }
       return undefined;
+    })
+    .catch((e) => {
+      if (e?.statusCode === 404) {
+        fastify.log.error(e);
+      }
+      return undefined;
```

Comment:
This will swallow non-404 errors that we probably still want to throw. Maybe instead of `return undefined;` we just want `throw e;` here?

Reply:
I'll test it myself when I get a chance, but what does the user see instead of the error page? Is it ok that the fetch silently fails? I wonder if we should get help from Berto verifying the fix (not sure of his GitHub username)

Reply:
The user shouldn't see an error page.. but if you want to test for the Elastic tile.. when you have the operator installed you shouldn't see `Disabled` in the title and when you don't have it installed.. there should be a test implying `Disabled` in the Elastic tile.